src/Dummy.hpp

```diff
@@ -1,19 +1,17 @@
 #ifndef _DUMMYPROJECT_DUMMY_HPP_
 #define _DUMMYPROJECT_DUMMY_HPP_
```

Comment:
Not standard (yet widely supported) and for problems with pragma once, have a look at

http://en.wikipedia.org/wiki/Pragma_once#Caveats